.stylelintrc

```diff
@@ -0,0 +1,160 @@
+{
+  "plugins": [
+    "stylelint-scss"
+  ],
+  "ignoreFiles": [
```

Comment:
We can but for some files (the ones I added here) it didn’t make sense. If you look into the specific files you’ll find already a couple of these inline command-comments but if it’s around the whole file (because the mixin/fn isn’t lintable by stylelint due to unrecognized assignments etc) this doesn’t make sense, I think.